5_resultados.tex

```diff
@@ -1,9 +1,133 @@
 \chapter{Resultados e análises}\label{cap:resultados}
 
-Descreva os resultados encontrados e análises propostas
+Nesta seção será apresentada os resultados obtidos e os processos necessários para a obtenção dos mesmos. Como visto no Capitulo 4 uma das fases necessárias para a Análise de Sentimento é a classificação de polaridade dos \textit{tweets}. Durante a classificação, que era o resultado da execução do Naive Bayes, foram utilizadas diversas formas de execução que serão apresentadas abaixo
```

Comment:
No começo do parágrafo, não seria "Neste capítulo"?

"Neste capítulo serÃO apresentados os resultados obtidos": tem que consertar a concordância

"Como visto no Capítulo 4": você tá falando do capítulo de referencial teórico? Se for, é o Capítulo 2.

"Como visto no Capitulo 4 uma das fases necessárias para a Análise de Sentimento é a classificação de polaridade dos \textit{tweets}." -> "Como visto anteriormente no Capítulo 2, uma das etapas da Mineração de Opinião" (se você reparar o capítulo 2, ele fala das etapas na seção de Mineração de Opinião e não Análise de sentimento)

Cara, tem um trecho que você escreve "Durante a classificação, que era o resultado da execução do Naive Bayes" eu não entendi o sentido da frase. Você quis dizer "gera" ao invés de "era"?